### PR TITLE
US104432: Update My Courses to version 8.

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

**Comment by njostonehouse** [comment]
Missed `d2l-resize-aware`, adding now.

File: package-lock.json [hunk 1]
```diff
@@ -3092,7 +3092,7 @@
       }
     },
     "d2l-table": {
-      "version": "github:BrightspaceUI/table#3de8bde8611d203acc0bdc86d7ffed26cb6c46a1",
+      "version": "github:BrightspaceUI/table#50eddad06ea70db8ac453ac78987e3fd42ec9de4",
```
Comment (edwardg8) [inline package-lock.json]: Do we want to update version of all of those `d2l-` web components?

Comment (njostonehouse) [reply package-lock.json]: I assume so. I've added the person who released it to be sure.

**Comment by njostonehouse** [comment]
Build is still failing. Closing to try again with fewer updates.

**Pull request closed without merging**